### PR TITLE
navigate user to product page when user isn't issuer

### PR DESCRIPTION
**Pull request by zip520123**

### CODE REVIEW

File: components/auth/Login.tsx [hunk 1]
```diff
@@ -122,23 +122,23 @@ const Login: FC = () => {
...
         userInfo: {
           email: account,
           password,
         },
       });
-      if (!issuer || issuer.length === 0) {
+      if (!user || user.length === 0) {
         notify.error({ msg: err || '此帳號並不存在' });
-      } else if (!issuer[0].isIssuer) {
+      } else if (!user[0].isIssuer) {
```
Comment (TseHang) [inline components/auth/Login.tsx]: I think this condition should be like `else if (mode ==== 'issuer' && !user[0].isIssuer)` ?

File: components/auth/Login.tsx [hunk 2]
```diff
@@ -122,23 +122,23 @@ const Login: FC = () => {
...
         notify.error({ msg: '此帳號並不是發證機關帳號，請確認使用帳號' });
       } else {
         updateUser({
-          ...preparedUser(issuer[0]),
-          loginMode: 'issuer',
+          ...preparedUser(user[0]),
+          loginMode: mode,
         });
-        Router.push('/issuer');
+        Router.push(user[0].isIssuer === true ? '/issuer' : '/product');
```
Comment (TseHang) [inline components/auth/Login.tsx]: Actually the condition equals to `user[0].isIssuer ? '/issuer' : '/product'`

File: pages/product.tsx [hunk 3]
```diff
@@ -71,4 +72,6 @@ ProductPage.getInitialProps = async () => ({
   namespacesRequired: [i18nNamespace.Common, i18nNamespace.Demo],
 });
 
-export default ProductPage;
+export default withAuth('user', () => {
+  return { namespacesRequired: [i18nNamespace.Common] };
```
Comment (TseHang) [inline pages/product.tsx]: Seems lake a `i18nNamespace.Demo` in namespaceRequired

**Comment by TseHang** [comment]
@zip520123   some comments, plz see and fix. thanks.

File: components/auth/Login.tsx (hunk 2 above)
Comment (zip520123) [reply components/auth/Login.tsx]: Yes, it is the same.

**Comment by TseHang** [comment]
LGTM

**Pull request merged**